### ISSUE TITLE
SDL_ttf.h

### ISSUE THREAD

**Issue by JeremiahCheatham** [foggynight/gforth-sdl2#1]
I have been attempting to use a modified version of these bindings in my game project. I have had to make some changes to get them working but i am stuck on the SDL_ttf.h binding.
I had to change the

    s" SDL2" add-lib

to

    s" SDL2_ttf" add-lib

I also had to comment out all of these functions to get it to compile however i need the function TTF_RenderText_Blended. I don't know if it's because it needs a SDL_Color struct or if it's not linking to the actual SDL library since this is not core SDL2. My project is here. https://github.com/JeremiahCheatham/Yellow-Snow/tree/main/Gforth-SDL2

\ c-function TTF_RenderText_Solid TTF_RenderText_Solid a a n -- a	( font text fg -- )
\ c-function TTF_RenderUTF8_Solid TTF_RenderUTF8_Solid a a n -- a	( font text fg -- )
\ c-function TTF_RenderUNICODE_Solid TTF_RenderUNICODE_Solid a a n -- a	( font text fg -- )
\ c-function TTF_RenderText_Solid_Wrapped TTF_RenderText_Solid_Wrapped a a n n -- a	( font text fg wrapLength -- )
\ c-function TTF_RenderUTF8_Solid_Wrapped TTF_RenderUTF8_Solid_Wrapped a a n n -- a	( font text fg wrapLength -- )
\ c-function TTF_RenderUNICODE_Solid_Wrapped TTF_RenderUNICODE_Solid_Wrapped a a n n -- a	( font text fg wrapLength -- )
\ c-function TTF_RenderGlyph_Solid TTF_RenderGlyph_Solid a n n -- a	( font ch fg -- )
\ c-function TTF_RenderGlyph32_Solid TTF_RenderGlyph32_Solid a n n -- a	( font ch fg -- )
\ c-function TTF_RenderText_Shaded TTF_RenderText_Shaded a a n n -- a	( font text fg bg -- )
\ c-function TTF_RenderUTF8_Shaded TTF_RenderUTF8_Shaded a a n n -- a	( font text fg bg -- )
\ c-function TTF_RenderUNICODE_Shaded TTF_RenderUNICODE_Shaded a a n n -- a	( font text fg bg -- )
\ c-function TTF_RenderText_Shaded_Wrapped TTF_RenderText_Shaded_Wrapped a a n n n -- a	( font text fg bg wrapLength -- )
\ c-function TTF_RenderUTF8_Shaded_Wrapped TTF_RenderUTF8_Shaded_Wrapped a a n n n -- a	( font text fg bg wrapLength -- )
\ c-function TTF_RenderUNICODE_Shaded_Wrapped TTF_RenderUNICODE_Shaded_Wrapped a a n n n -- a	( font text fg bg wrapLength -- )
\ c-function TTF_RenderGlyph_Shaded TTF_RenderGlyph_Shaded a n n n -- a	( font ch fg bg -- )
\ c-function TTF_RenderGlyph32_Shaded TTF_RenderGlyph32_Shaded a n n n -- a	( font ch fg bg -- )
\ c-function TTF_RenderText_Blended TTF_RenderText_Blended a a n -- a	( font text fg -- )
\ c-function TTF_RenderUTF8_Blended TTF_RenderUTF8_Blended a a n -- a	( font text fg -- )
\ c-function TTF_RenderUNICODE_Blended TTF_RenderUNICODE_Blended a a n -- a	( font text fg -- )
\ c-function TTF_RenderText_Blended_Wrapped TTF_RenderText_Blended_Wrapped a a n n -- a	( font text fg wrapLength -- )
\ c-function TTF_RenderUTF8_Blended_Wrapped TTF_RenderUTF8_Blended_Wrapped a a n n -- a	( font text fg wrapLength -- )
\ c-function TTF_RenderUNICODE_Blended_Wrapped TTF_RenderUNICODE_Blended_Wrapped a a n n -- a	( font text fg wrapLength -- )
\ c-function TTF_RenderGlyph_Blended TTF_RenderGlyph_Blended a n n -- a	( font ch fg -- )
\ c-function TTF_RenderGlyph32_Blended TTF_RenderGlyph32_Blended a n n -- a	( font ch fg -- )
\ c-function TTF_RenderText_LCD TTF_RenderText_LCD a a n n -- a	( font text fg bg -- )
\ c-function TTF_RenderUTF8_LCD TTF_RenderUTF8_LCD a a n n -- a	( font text fg bg -- )
\ c-function TTF_RenderUNICODE_LCD TTF_RenderUNICODE_LCD a a n n -- a	( font text fg bg -- )
\ c-function TTF_RenderText_LCD_Wrapped TTF_RenderText_LCD_Wrapped a a n n n -- a	( font text fg bg wrapLength -- )
\ c-function TTF_RenderUTF8_LCD_Wrapped TTF_RenderUTF8_LCD_Wrapped a a n n n -- a	( font text fg bg wrapLength -- )
\ c-function TTF_RenderUNICODE_LCD_Wrapped TTF_RenderUNICODE_LCD_Wrapped a a n n n -- a	( font text fg bg wrapLength -- )
\ c-function TTF_RenderGlyph_LCD TTF_RenderGlyph_LCD a n n n -- a	( font ch fg bg -- )
\ c-function TTF_RenderGlyph32_LCD TTF_RenderGlyph32_LCD a n n n -- a	( font ch fg bg -- )

**Comment by foggynight** [comment]
Hello, sorry for taking so long to get back to you, haven't been at my computer much lately. Did you get this figured out after? I was having all kinds of trouble getting these bindings working, I feel your pain!

**Comment by JeremiahCheatham** [comment]
Yes i was able to just modify everything over a couple weeks or month and came up with these bindings.

https://github.com/JeremiahCheatham/Gforth-SDL2-Bindings

And this complete working SDL2 Demo with everything you need.

https://github.com/JeremiahCheatham/SDL2-Examples

And this Game.

https://github.com/JeremiahCheatham/Yellow-Snow
